tests/records/dumpers/test_access_dumpers.py

```diff
@@ -10,25 +10,24 @@
 import pytest
 from invenio_records.dumpers import ElasticsearchDumper
 
-from invenio_rdm_records.records import RDMDraft, RDMRecord
+from invenio_rdm_records.records import RDMDraft, RDMParent, RDMRecord
 from invenio_rdm_records.records.dumpers import GrantTokensDumperExt
 from invenio_rdm_records.records.systemfields.access import Grant
 from invenio_rdm_records.services import RDMRecordService
 
 
-def test_grant_tokens_dumper(app, db, minimal_record):
+def test_grant_tokens_dumper(app, db, minimal_record, location):
     """Test grant token dumper extension implementation."""
     dumper = ElasticsearchDumper(
         extensions=[GrantTokensDumperExt("access.grant_tokens")]
     )
 
+    # Create the record
     minimal_record["access"]["grants"] = [
         {"subject": "user", "id": "1", "level": "view"},
         {"subject": "user", "id": "2", "level": "manage"},
     ]
-
```

Comment:
Comment: ``RDMDraft`` is responsible for creating the parent record, so when we use ``RDMRecord`` directly without a draft, we have manually create it. Now, it can make sense to use ``RDMRecord`` because it also doesn't create buckets etc, hence no need for a files location.